src/malli/generator.cljc

```diff
@@ -412,11 +427,21 @@
 (defmethod -schema-generator :< [schema options] (-double-gen {:max (-> schema (m/children options) first dec)}))
 (defmethod -schema-generator :<= [schema options] (-double-gen {:max (-> schema (m/children options) first)}))
 (defmethod -schema-generator := [schema options] (gen/return (first (m/children schema options))))
-(defmethod -schema-generator :not= [schema options] (gen/such-that #(not= % (-> schema (m/children options) first)) gen/any-printable 100))
+(defmethod -schema-generator :not= [schema options] (gen/such-that #(not= % (-> schema (m/children options) first)) gen/any-printable
+                                                                   {:max-tries 100
```

Comment:
some of these have `:max-tries`, but not all, is there a reason for this?

Reply:
My best guess is that the ones without `:max-tries` always succeed in practice.